src/propName.js

```diff
@@ -6,12 +6,9 @@ export default function propName(prop = {}) {
     throw new Error('The prop must be a JSXAttribute collected by the AST parser.');
   }
 
-  switch (prop.name.type) {
-    case 'JSXIdentifier':
-      return prop.name.name;
-    case 'JSXNamespacedName':
-      return `${prop.name.namespace.name}:${prop.name.name.name}`;
-    default:
-      return undefined;
+  if (prop.name.type === 'JSXNamespacedName') {
+    return `${prop.name.namespace.name}:${prop.name.name.name}`;
   }
+
+  return prop.name.name;
```

Comment:
This could cause upstream failures if `prop.name` is undefined. Do we want to keep the switch statement and return an empty string, or do you think it's ok for an error to be thrown (i'm indifferent, its an edge case)